cluster-autoscaler/Makefile

```diff
@@ -49,7 +51,7 @@ make-image: make-image-arch-$(GOARCH)
 
 make-image-arch-%:
 ifdef BASEIMAGE
-	docker build --pull --build-arg BASEIMAGE=${BASEIMAGE} \
```

Comment:
By removing --pull, the local baseimage will be used. This allows one to create a local image of the jujucli, and use that as a baseimage of the autoscaler image without having to host the juju cli image in a registry somewhere.